glTFLoader/Interface.cs

```diff
@@ -19,16 +14,15 @@ public static Gltf LoadModel(string filePath)
             var path = Path.GetFullPath(filePath);
             CallContext.LogicalSetData("UriRootPath", Path.GetDirectoryName(path));
             var bytes = File.ReadAllBytes(path);
```

Comment:
Should consider changing this to use a stream with a binary reader. There isn't any point to reading all of the bytes if only the JSON will be used.